Transfer/Sources/CreateTransferController.swift

```diff
@@ -43,6 +43,7 @@ final class CreateTransferController: UITableViewController {
         (TransferButtonCell.self, TransferButtonCell.reuseIdentifier),
         (TransferNotesCell.self, TransferNotesCell.reuseIdentifier)
     ]
+    private var buttonSelection: Bool = false
```

Comment:
No need of adding this. Move the handling of error.group != .business in presenter#createInitialTransfer()

Reply:
ok